scripts/release.sh

```diff
@@ -102,7 +102,7 @@ main() {
 
   # Check go version.
   local go_version current_go_version
-  go_version="go$(run_go_tool "github.com/mikefarah/yq/v3" read .github/workflows/build.yaml "jobs.build.steps[1].with.go-version")"
+  go_version="go$(grep go-version .github/workflows/build.yaml | awk '{print $2}' | tr -d '"')"
```

Comment:
I was not able to manage to get the go tool to run in CI. It always failed saying that dependency is missing. It would be even harder if I wanted to use this approach to support old release branches.